### ISSUE TITLE
Fortran solverdummy in 'Bindings/Solverdummy' test fails

### ISSUE THREAD

**Issue by Eder-K** [precice/systemtests#214]
See [this example build](https://travis-ci.org/github/precice/systemtests/jobs/673534199#L1515):
```
Step 23/35 : RUN make
 ---> Running in b799119bd02c
make: *** No targets specified and no makefile found.  Stop.
```
Affects both the 16.04 and 18.04 version of the test. The issue stems from a recent update to the solverdummies in https://github.com/precice/precice/pull/702.

**Comment by MakisH** [comment]
https://github.com/precice/precice/pull/702/files#diff-d8797295ea536ba0faa48fcecaf72a97 removed the included Makefile and added CMake. So, we first need to configure it now.

**Comment by Eder-K** [comment]
Recently fixed in #216 by adding the needed cmake command.
